docs/index.md

```diff
@@ -6,6 +6,13 @@ This site provides information on open source projects that Ripple engineers are
 
 While Ripple may be the primary developer on these open source projects, it is not a centralized authority (there isn't one) and  there is no central actor. Community contributions and feedback are welcome!
 
-Proposals for new features and amendments go through the consensus process for review and approval by the XRP Ledger community before they can be merge into the XRP Ledger code base. Once a feature is merged on the `master` branch of the `rippled` codebase, documentation for that feature will be available on [xrpl.org](https://xrpl.org).
+Proposals for new features and amendments go through the consensus process for review and approval by the XRP Ledger community before they can be merged into the XRP Ledger code base. Once a feature is merged on the `master` branch of the `rippled` codebase, documentation for that feature will be available on [xrpl.org](https://xrpl.org).
```

Comment:
Proposals for new features and amendments are reviewed and approved through the consensus process by the XRP Ledger community before they are merged into the XRP Ledger code base. Once a feature is merged into the `master` branch of the `rippled` code base, documentation for that feature is available on [xrpl.org](https://xrpl.org).

[At a minimum, choose whether _code base_ is one or two words.]